src/Api/PubnubApi/EndPoint/Access/AuditOperation.cs

```diff
@@ -80,6 +80,16 @@ public void Execute(PNCallback<PNAccessManagerAuditResult> callback)
 #endif
         }
 
+
+        public async Task<PNResult<PNAccessManagerAuditResult>> ExecuteAsync()
+        {
+#if NETFX_CORE || WINDOWS_UWP || UAP || NETSTANDARD10 || NETSTANDARD11 || NETSTANDARD12
+            return await AuditAccess(this.channelName, this.channelGroupName, this.authenticationKeys, this.queryParam).ConfigureAwait(false);
+#else
+            return await AuditAccess(this.channelName, this.channelGroupName, this.authenticationKeys, this.queryParam).ConfigureAwait(false);
```

Comment:
This `else` statement looks the same as the `if` am I missing something here or are you? :-P

Reply:
will remove if else